test/integration/checkout_int_test.go

```diff
@@ -262,6 +262,28 @@ func (suite *CheckoutIntegrationTestSuite) TestCheckoutCaseInsensitive() {
 	suite.Assert().NotContains(string(bytes), "ACTIVESTATE-CLI/SMALL-PYTHON", "kept incorrect namespace case")
 }
 
+func (suite *CheckoutIntegrationTestSuite) TestCheckoutBuildtimeClosure() {
+	suite.OnlyRunForTags(tagsuite.Checkout)
+
+	if runtime.GOOS != "linux" {
+		suite.T().Skip("Skipping buildtime closure test on non-linux platform")
+	}
+
+	ts := e2e.New(suite.T(), false)
+	defer ts.Close()
+
+	cp := ts.SpawnWithOpts(
+		e2e.OptArgs("checkout", "ActiveState-CLI/small-python"),
```

Comment:
We should probably checkout with a UUID. Otherwise this may fail as the small-python project is updated.

Reply:
Good point!